go.mod

```diff
@@ -9,10 +9,12 @@ require (
 	github.com/openebs/lvm-localpv v0.6.0
 	github.com/openebs/zfs-localpv v1.8.0
 	github.com/pkg/errors v0.9.1
+	github.com/ramya-rao-a/go-outline v0.0.0-20210608161538-9736a4bde949 // indirect
```

Comment:
This file should not have been changed.

Reply:
You might need to rebase your code against `develop` branch, this library isn't required by any change you've done IMO.